azure-pipelines.yaml

```diff
@@ -41,7 +43,8 @@ jobs:
           inputs:
             version: $(DotNetVersion)
         - pwsh: |
-            dotnet workload install android
+            dotnet workload update --verbosity diag --from-rollback-file workload.json --source $(Dotnet6Source) --source $(NuGetOrgSource)
```

Comment:
so this is trick to "pin" versions for workloads? 

`workload.json` is default name or can it be something else?